src/ui/UserProfile.cpp

```diff
@@ -106,6 +125,12 @@ UserProfile::avatarUrl()
         return cache::avatarUrl(roomid_, userid_);
 }
 
+bool
+UserProfile::globalUserProfile() const
+{
+        return (roomid_ == "") && isSelf();
```

Comment:
I guess you should remove the isSelf here too, if you want to be able to view the global user profile of any user.